src/components/Navbar/constants.tsx

```diff
@@ -0,0 +1,74 @@
+import { Trans } from '@lingui/macro'
+import ExternalLink from 'components/ExternalLink'
+import { CSSProperties } from 'react'
+
+type ResourceItem = {
+  label: JSX.Element
+  key: string
+  style: CSSProperties
+}
+
+export const resourcesMenuItems = (): ResourceItem[] => {
+  return [
+    {
+      key: 'governance',
+      label: (
```

Comment:
This broke because `MenuItemType` has changed is a recent `antd` update ( https://ant.design/components/menu/#MenuItemType)